### ISSUE TITLE
Evaluating new HVAC efficiencies 

### ISSUE THREAD

**Issue by edwardv720** [canmet-energy/btap_tasks#196]
### Description
The TG-HVAC and SWH are considering new HVAC efficiency standard. These standards are expected to match NRCan's energy efficiency regulations.

### Approach
These new efficiencies will be evaluated by means of a special branch of openstudio-standards (nrcan-nrc); the proposed efficiencies will be entered into the data json files. A measure isn't used because the efficiencies require a sizing run to be done and must follow current BTAP rules/assumptions in prescribing efficiencies.  

### Repositories Involved
https://github.com/NREL/openstudio-standards/tree/nrcan
https://github.com/edwardv720/openstudio-standards/tree/nrcan_196

**Comment by IainMacdonald** [comment]
@edwardv720 which json data do you need to change. @phylroy has a different plan and will talk us through it next week (include original and updated json file).

**Comment by edwardv720** [comment]
The following json files contain all the hvac data currently used for 2015 NECB archetypes
- In NECB2015: boilers, furnaces, heat_pumps, heat_pumps_heating, unitary_acs
- NECB 2011: chillers, heat rejections

**Comment by phylroy** [comment]
Waiting on JSON zipped file.

**Comment by edwardv720** [comment]
[NECB 2015 new eff.zip](https://github.com/canmet-energy/btap_tasks/files/2654688/NECB.2015.new.eff.zip)

**Comment by phylroy** [comment]
@edwardv720 Are you updating the entire tables?

**Comment by edwardv720** [comment]
Just the efficiency (e.g. EER) and I changed the limits (the new standards have put max cap at 223kw for  in heat_pumps.json

**Comment by phylroy** [comment]
okay and you just want to apply this to 2015 buildings?

**Comment by edwardv720** [comment]
The plan is to apply these to 2017 once that's ready.

**Comment by phylroy** [comment]
We can do that internally with the 2017 and provide you the results. This shouldn't take to long to write measure to do. I'll start on this and your envelope run that you submitted on Monday.

**Comment by edwardv720** [comment]
Ok thank you. 
Just to confirm, you're referring to using mariana's envelope measures on 2017 right? (I don't think we sent you anything on Monday)

**Comment by phylroy** [comment]
@IainMacdonald what building do you want simulated?

**Comment by IainMacdonald** [comment]
@phylroy not sure. How about all archetypes in each of the climate zones (I think that's just over 100 runs). If you can do the simulations we can write the report.

**Comment by IainMacdonald** [comment]
@phylroy no rush... just give me a date that works for you!

**Comment by phylroy** [comment]
@IainMacdonald  Added side loading feature to btap_create_prototype measure in this branch. 

https://github.com/canmet-energy/btap/tree/nrcan_196

Things to know...
* You put the only the  tables that you want sideloaded into the resources/data_sideload folder. It will automatically side load the information. 

* Only tables are supported right now, not functions or constants .. and it will over-write the previous standard table in memory completely.. so no partial updates.  Looks like your task fits into this. 

*  The template value in the json data must match the template you are exploring. So I had to rename all your template key values in your json files to NECB2017.  This restriction should really be removed by us....maybe later. But this means you can only run NECB2017. 

@edwardv720 I have added your json data to the resources/data_sideload in this branch. if you are satisfied with the output, create a PAT project using this measure.  I'll run it on all the building and locations once you send it over and send you the results.  

If this test  works out I will green light this to be merged into master later.

**Comment by phylroy** [comment]
@waiting on @edwardv720 to validate measure above

**Comment by IainMacdonald** [comment]
@phylroy this task is not required anymore but the side loading measure is probably a useful addition to BTAP. @edwardv720 says that there is one test case. Will that suffice? How do you want to progress? I'm concerned that with one test case we could be missing something but don't want 20,000 test cases either!
